docs/application/native/guides/location-sensors/user-awareness.md

```diff
@@ -0,0 +1,155 @@
+    ```
+    ua_monitor_start_presence_detection(g_ua_mon_h, NULL, detection_mode, __presence_detected_cb, NULL);
+    ```
+    The ```__presence_detected_cb()``` function is a callback, which is called when the user presence is detected.
+  
+2.  Using the user presence detection consumes the system resources, so if the service is not used, stop the user presence detection using the ```ua_monitor_stop_presence_detection()``` API. Call the ```ua_monitor_start_presence_detection()``` API again if user presence detection is needed.
+    ```
+    ua_monitor_stop_presence_detection(g_ua_mon_h);
+    ```
+  
+## Start the absence detection
+
+To start the absence detection, follow these steps:
+  
+1.  Start the user absence detection using the `ua_monitor_start_absence_detection()` API. This call is asynchronous and only initiates the process of starting the user absence detection. Once the absence detection is started, the registered callbacks are invoked when their corresponding events take place. To know when the user absence is detected, use the `ua_absence_detected_cb()` function:
+    ```
+    ua_monitor_start_absence_detection(g_ua_mon_h, NULL, detection_mode, __absence_detected_cb, NULL);
+    ```
+    The `__absence_detected_cb()` is a callback function, which is called when the user absence is detected. 
+    
+2.  Using the user absence detection consumes the system resources, so if the service is not used, stop the user absence detection using the `ua_monitor_stop_absence_detection()` API. Call the `ua_monitor_start_absence_detection()` API again if user presence detection is needed:
+    ```
+    ua_monitor_stop_absence_detection(g_ua_mon_h);
+    ```
+ 
+  
+## Start the location detection
+
+To start the location detection, follow these steps:
+
+1.  Start the user location detection using the `ua_monitor_start_location_detection()` API. This call is asynchronous and only initiates the process of starting the user location detection. Once the location detection is started, the registered callbacks are invoked when their corresponding events take place. To know when the user location is detected, use the `ua_location_detected_cb()` function:
+    ```
+    ua_monitor_start_location_detection(g_ua_mon_h, NULL, detection_mode, __location_detected_cb, NULL);
+    ```
+    The `__location_detected_cb()` is a callback function, which is called when the user location is detected.
+    
+2.  Using the user location detection consumes the system resources, so if the service is not used, stop the `ua_monitor_stop_location_detection()` API. Call the `ua_monitor_start_location_detection()` API again if user location detection is needed: 
```

Comment:
Just a question to this update : https://github.com/Samsung/tizen-docs/pull/1713#discussion_r1011638649

**Original sentence :** stop the user location detection using the `ua_monitor_stop_location_detection()` API
**Suggestion :** stop the `ua_monitor_stop_location_detection()` API

In my understanding in the original sentence, `ua_monitor_stop_location_detection()` is used to stop the user location detection, 
but in suggestion `ua_monitor_stop_location_detection()` it self is stopped.
So I'm a little be confused. Is this update has the same meaning with the original sentence?
The sample code is also seems calling `ua_monitor_stop_location_detection()` api, to stop the service. 
Would you please check?

Reply:
@mijin85cho , thank you for pointing out the issue. I understand that it effects the meaning of the sentence. Therefore, I'm reverting the suggested changes back to it's original text.